include/avif/avif.h

```diff
@@ -602,9 +578,26 @@ AVIF_API uint32_t avifRGBImagePixelSize(const avifRGBImage * rgb);
 AVIF_API void avifRGBImageAllocatePixels(avifRGBImage * rgb);
 AVIF_API void avifRGBImageFreePixels(avifRGBImage * rgb);
 
-// The main conversion functions
-AVIF_API avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb);
-AVIF_API avifResult avifImageYUVToRGB(const avifImage * image, avifRGBImage * rgb);
+// The main conversion options.
+typedef enum avifConversionFlag
+{
+    // Conversion between RGB and YUV samples:
+    AVIF_CONVERSION_AUTO = 0,                // prefers libyuv if available (default)
+    AVIF_CONVERSION_AVOID_LIBYUV = (1 << 0), // only picks built-in paths
+
+    // Conversion from RGB to YUV (ignored when converting from RGB to YUV), set at most one:
```

Comment:
In the parentheses, change "from RGB to YUV" to "from YUV to RGB".

These flags (currently only one) specify the chroma downsampling filter, so the comment should start a new subsection for them, e.g.,

```
    // Conversion from RGB to YUV (ignored when converting from YUV to RGB)

    // Chroma downsampling filter. Set at most one:
```

We should say that they are ignored if the conversion does not need to downsample chroma (when output is YUV 4:4:4 or YUV 4:0:0). We shoudl also say what is the default chroma downsampling filter when none is specified.